src/common/types.hpp

```diff
@@ -344,6 +376,14 @@ class Ip6Prefix
      */
     bool IsValid(void) const { return mLength > 0 && mLength <= 128; }
```

Comment:
Do we need to update this? We're now having prefixes with zero length.

Reply:
sounds reasonable! but this is not in the scope of this change. I can create another commit to fix this.